src/main/java/com/myaudiolibrary/web/controller/ArtistController.java

```diff
@@ -0,0 +1,88 @@
+package com.myaudiolibrary.web.controller;
+
+import com.myaudiolibrary.web.model.Artist;
+import com.myaudiolibrary.web.repository.ArtistRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import javax.persistence.EntityExistsException;
+import javax.persistence.EntityNotFoundException;
+
+import static org.springframework.http.HttpStatus.NO_CONTENT;
+
+@RestController
+@RequestMapping(value = "/artists")
+public class ArtistController {
+
+    private final String ERROR_ARTIST_NOT_FOUND = "L'artiste demandé n'existe pas !";
+    private final String ERROR_NAME_EMPTY = "Le nom passé en paramètre est vide !";
+
+    @Autowired
+    private ArtistRepository artistRepository;
+
+    @RequestMapping(method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE, value = "/{idArtist}")
+    public Artist getArtiste(@PathVariable(value = "idArtist") Long idArtist) {
+        Artist artist = artistRepository.findById(idArtist).orElse(null);
+        if(artist != null) {
+            return artist;
+        } else {
+            throw new EntityNotFoundException(ERROR_ARTIST_NOT_FOUND);
+        }
+    }
```

Comment:
Ce n'est pas idéal d'utiliser le `Optional` de cette manière
```suggestion
    public Optional<Artist> getArtiste(@PathVariable(value = "idArtist") Long idArtist) {
        Optional<Artist> artist = artistRepository.findById(idArtist);
        if(!artist.isEmpty()) {
            return artist;
        } 
        throw new EntityNotFoundException(ERROR_ARTIST_NOT_FOUND);
    }
```